projects/cashmere/src/lib/sass/dialog.scss

```diff
@@ -0,0 +1,64 @@
+$hc-dialog-padding: 24px !default;
+$hc-dialog-border-radius: 4px !default;
+$hc-dialog-max-height: 65vh !default;
+$hc-dialog-button-margin: 8px !default;
+
+@mixin hc-dialog-container() {
+    border-radius: $hc-dialog-border-radius;
+    box-sizing: border-box;
+    display: block;
+    outline: solid 1px;
```

Comment:
We previously had two `outline` rules on this element. I kept the latter one. @benjanderson I seem to remember you were the last one to work on this, are you aware of any reason we might want to keep both?